addons/common/functions/fnc_loadSettingsLocalizedText.sqf

```diff
@@ -15,17 +15,16 @@
 private ["_parseConfigForDisplayNames", "_name"];
 
 _parseConfigForDisplayNames = {
-    private "_optionEntry";
+    private ["_optionEntry", "_values", "_text"];
     _optionEntry = _this select 0;
     if !(isClass _optionEntry) exitwith {false};
+    _values = getArray (_optionEntry >> "values");
     _x set [3, getText (_optionEntry >> "displayName")];
     _x set [4, getText (_optionEntry >> "description")];
+    _x set [5, _values];
     _x set [8, getText (_optionEntry >> "category")];
 
-    private "_values";
-    _values = _x select 5;
     {
-        private "_text";
         _text = _x;
         if (((typeName _text) == "STRING") && {(count _text) > 1} && {(_text select [0,1]) == "$"}) then {
```

Comment:
Because of the change on line 21;
```sqf
_values = getArray (_optionEntry >> "values");
```
We no longer require to be looping through the values and localize each value.